app/test/unit/entities/areaV2.entity.test.js

```diff
@@ -615,5 +616,407 @@ describe('Area Entity V2', () => {
                 });
             });
         });
+
+        describe('Using Admin Versions to Populate Area Admin Information', () => {
+            context('Complete Administrative Boundary Info', () => {
+                const areaData = {
+                    name: 'Distrito Central, Francisco Morazán, Honduras',
+                    geostore: 'abcf7041e2fbc5e8e7774178157ababe',
+                    iso: {
+                        country: 'HND',
+                        region: '8',
+                        subregion: '4',
+                    },
+                    admin: {
+                        adm0: 'HND',
+                        adm1: 8,
+                        adm2: 4,
+                    },
+                    adminVersions: [
+                        {
+                            provider: 'gadm',
+                            version: '3.6',
+                            geostore: '923f7035e2fbc5e8e6134178157abab4',
+                            country: { id: 'KEN', name: 'Kenya' },
```

Comment:
I think I'm missing something - should it be possible to have this mismatch between the values in the `admin` and `iso` objects of the input area and those in the  `adminVersions` record?

Reply:
Great question, @solomon-negusse! I should have clarified my intent better in the test suite.
I picked such extremes to prove that the `iso` and `admin` are updated when `populateAdminInfo` is called. I wanted very different values so it was absolutely clear that the `adminVersion` was being used.

There won't be these extremes in GADM 3.6 production behavior. I'd expect them to be identical, actually. But it would be really hard to prove the behavior in a test.